src/backend/services/wallet-api/src/routes.py

```diff
@@ -27,7 +28,15 @@ def createWallet():
     if (data is None) or ("walletId" not in data):
         raise IncorrectPayloadException()
     
-    response = requests.post( blockchain_wallet_url.format("addWallet"), json=data)
+    walletId = data["walletId"]
+    response = send_get_request( user_api_url.format("umnetID/"+ walletId), None)
```

Comment:
Should this be using umnetID instead of walletId for this call?

Reply:
I think we mentioned that the walletids will be umnetids from now. Will require changes in the frontend as well

Reply:
Ah yes right, I think we should rename the payload fields to umnetId from walletId then, and make the corresponding frontend and backend changes!

Reply:
I am on the frontend changes, I think we should still keep `walletId` as the name. I can see walletId being something else from umnetId and that would require the changes again. will help abstracting things out a little